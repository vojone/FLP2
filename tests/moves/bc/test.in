123
456
789
444 111 123 333
111 222 456 444
111 222 789 444
123
662
661
