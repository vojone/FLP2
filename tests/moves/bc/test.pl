:- consult('../../../cube').

:- read_cube(C, _), move_b_ccw(C, _, _, RC), write_cube(RC).
