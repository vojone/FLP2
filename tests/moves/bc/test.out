443
456
789
444 111 369 633
111 222 258 644
111 223 147 144
123
662
221
